lawnchair/src/app/lawnchair/LawnchairLauncher.kt

```diff
@@ -151,38 +169,15 @@ class LawnchairLauncher : QuickstepLauncher(), LifecycleOwner,
             }
         }
     }
-    private val _onBackPressedDispatcher = OnBackPressedDispatcher {
-        super.onBackPressed()
-    }
+    override val onBackPressedDispatcher = OnBackPressedDispatcher()
+    override val lifecycle: LifecycleRegistry = LifecycleRegistry(this)
```

Comment:
Declare the override type as `LifecycleRegistry` instead of `Lifecycle` to inline the `lifecycleRegistry` field before.